### ISSUE TITLE
Zombie Awareness's scent entities interfere with Thaumcraft aspect scanning of items

### ISSUE THREAD

**Issue by Barhandar** [Haggle1996/RevolutionPack#546]
Solution: please add [Thaumcraft Inventory Scanning](https://minecraft.curseforge.com/projects/thaumcraft-inventory-scanning) to the pack.

**Comment by Shuro** [comment]
Totally agree on that, it should be included. A nice mod.

**Comment by aurumvorax** [comment]
I agree aout Inventory scanning, but I would also suggest disabling ZA scent, as I've had problems with the scent entities interfering with chromaticraft structures, similar to railcraft heat blocks
